News article for OTM-05 (#46)
dd-ix/website-content: content/news/018.open-tech-meetup-5.de.md

---
published: 2026-05-13
description: White-Boxes, Forschungsnetze, Flow Collectoren, einige neue Peers und leckeres Essen
title: Das 5. DD-IX Open Tech Meetup
keywords: [ Event ]
authors: [ Maurice ]
image: 018_open_tech_meetup_5_1.webp
---

> Das fünfte **Open Tech Meetup** fand am 07. Mai in der [**Brauerei Lohrmanns**](https://lohrmannsbrew.de/) im Kraftwerk Mitte statt.

Reviewer: ```suggestion
Das fünfte **Open Tech Meetup** fand am 07. Mai bei der [**Brauerei Lohrmanns**](https://lohrmannsbrew.de/) im Kraftwerk Mitte statt.
```

> Bisher war dies das größte OTM mit 56 Teilnehmern. Wir starteten mit technischen Problemen und gingen glücklicherweise dann über zu mehreren sehr interessanten Vorträgen.

Reviewer 2: ```suggestion
Bisher war dies unser größtes OTM mit 56 Teilnehmern. Wir starteten mit technischen Problemen und gingen glücklicherweise dann über zu mehreren sehr interessanten Vorträgen.
```

> Bisher war dies das größte OTM mit 64 angemeldeten und 56 erschienen Teilnehmern. Wir starteten mit technischen Problemen und gingen glücklicherweise dann über zu mehreren sehr interessanten Vorträgen.

Reviewer: ```suggestion
Bisher war dies das größte OTM mit 56 Teilnehmenden. Wir starteten mit technischen Problemen und gingen glücklicherweise dann über zu mehreren sehr interessanten Vorträgen.
```

> Im ersten Vortrag von Christian Dieckhoff mit dem Titel "White Box Early Adopter - What could possibly go wrong?" wurde viel über die Hürden beim Einsatz noch sehr neuer Software für Routing berichtet. Mit Geduld, viel Debugging und schnellen fixes vom Hersteller, wurden ein Großteil der Probleme aber schon gelöst.

Reviewer 2: ```suggestion
Im ersten Vortrag von Christian Dieckhoff mit dem Titel "White Box Early Adopter - What could possibly go wrong?" wurde viel über die Hürden beim Einsatz noch sehr neuer Software für Routing berichtet. Mit Geduld, viel Debugging und schnellen Fixes vom Hersteller wurden ein Großteil der Probleme aber schon gelöst.
```

> Im ersten Vortrag von Christian Dieckhoff mit dem Titel "White Box Early Adopter - What could possibly go wrong?" wurde viel über die Hürden beim Einsatz noch sehr neuer Hardware-Hersteller Software für Routing berichtet. Mit Geduld, viel Debugging und schnellen fixes vom Hersteller, wurden ein Großteil der Probleme aber schon gelöst.

Reviewer: ```suggestion
Im ersten Vortrag von Christian Dieckhoff mit dem Titel "White Box Early Adopter - What could possibly go wrong?" wurde viel über die Hürden beim Einsatz noch sehr neuer Software für Routing berichtet. Mit Geduld, viel Debugging und schnellen fixes vom Hersteller, wurden ein Großteil der Probleme aber schon gelöst.
```

![Vortrag White-Boxes](018_open_tech_meetup_5_1.webp)

> Danach berichtete Sebastian Neuner, wie in Badem-Würtemberg gerade ein Forschungsnetz aufgebaut wird. Wie sieht das mit Ausschreibungen aus, wie wurden die PoPs ausgewählt, welche Technologien werden genutzt und wer macht Remote Hands. Alles Fragen, in die wir einen Einblick erhalten haben.

Reviewer 2: ```suggestion
Danach berichtete Sebastian Neuner, wie in Baden-Würtemberg gerade ein Forschungsnetz aufgebaut wird: Wie sieht das mit Ausschreibungen aus, wie wurden die PoPs ausgewählt, welche Technologien werden genutzt und wer macht Remote Hands? Alles Fragen, in die wir einen Einblick erhalten haben.
```

> Danach ging es dem Thema, wie in Badem-Würtemberg gerade ein Forschungsnetz aufgebaut wird von Sebastian Neuner. Wie sieht das mit Ausschreibungen aus, wie wurden die PoPs ausgewählt, welche Technologien werden genutzt und wer macht Remote Hands. Alles Fragen, in die wir einen Einblick erhalten haben.

Reviewer: ```suggestion
Danach berichtete Sebastian Neuner, wie in Badem-Würtemberg gerade ein neues Forschungsnetz aufgebaut wird. Wie sieht das mit Ausschreibungen aus, wie wurden die PoPs ausgewählt, welche Technologien werden genutzt und wer macht Remote Hands. Alles Fragen, in die wir einen Einblick erhalten haben.
```

![Vortrag BelWü](018_open_tech_meetup_5_2.webp)

> Es ging weiter mit Gregor Düster und Jonas Mücke über "Deploying an IPFIX exporter in a university network". Wir erfuhren, wie man mit wenigen Ressourcen die Flow Daten der TU Dresden verarbeitet werden und einen Einblick in ein paar Datenauswertungen gab es auch. Mit einer spontanen Anfrage von ALASCA kam noch Friedrich Zahn, welcher SCOMB vorgestellt hat.

Reviewer 2: ```suggestion
Es ging weiter mit Gregor Düster und Jonas Mücke über "Deploying an IPFIX exporter in a university network". Wir erfuhren, wie mit wenigen Ressourcen die Flow Daten der TU Dresden verarbeitet werden. Zudem gab es einen Einblick in ein paar Datenauswertungen. Mit einer spontanen Anfrage von ALASCA kam noch Friedrich Zahn, welcher SCOMB vorgestellt hat.
```

> Es ging weiter mit Gregor Düster und Jonas Mücke über "Deploying an IPFIX exporter in a university network". Wir erfuhren, wie man mit wenigen Ressourcen die Flow Daten der ganzen Uni verarbeitet und einen Einblick in ein paar Datenauswertungen gab es auch. Mit einer spontanen Anfrage von ALASCA kam noch Friedrich Zahn, welcher SCOMB vorgestellt hat.

Reviewer: ```suggestion
Es ging weiter mit Gregor Düster und Jonas Mücke über "Deploying an IPFIX exporter in a university network". Wir erfuhren, wie man mit wenigen Ressourcen die Flow Daten der TU Dresden verarbeitet werden und einen Einblick in ein paar Datenauswertungen gab es auch. Mit einer spontanen Anfrage von ALASCA kam noch Friedrich Zahn, welcher SCOMB vorgestellt hat.
```

> Die Updates vom DD-IX haben wir diesmal kurz gehalten. Wir haben dank der **WOBCOM** unser BCIX Outreach auf **10G** aufstocken können. Außerdem haben wir unseren neuen Peers HLKomm und WOBCOM begrüßt. 

Reviewer 2: ```suggestion
Die Updates vom DD-IX haben wir diesmal kurz gehalten. Wir haben dank der **WOBCOM** unser BCIX Outreach auf **10G** aufstocken können. Außerdem haben wir unsere neuen Peers HLKomm und WOBCOM begrüßt. 
```

> Die Updates vom DD-IX haben wir diesmal kurz gehalten. Wir haben dank einem einem Peer unser BCIX Outreach auf **10G** aufstocken können. Außerdem haben wir unseren neuen Peers HLKomm und WOBCOM begrüßt. 

Reviewer: ```suggestion
Die Updates vom DD-IX haben wir diesmal kurz gehalten. Wir haben dank der WOBCOM unser BCIX Outreach auf **10G** aufstocken können. Außerdem haben wir unseren neuen Peers HLKomm und WOBCOM begrüßt. 
```

> Nach den Vorträgen beendeten wir den Abend noch mit Essen und Trinken, welches dank einem Sponsoring von **Flexoptics** kostenlos für alle Teilnehmer der Veranstaltung war. Herzlichen Dank nochmal!

Reviewer 2: ```suggestion
Nach den Vorträgen beendeten wir den Abend noch mit Essen und Trinken, welches dank einem Sponsoring von **Flexoptics** kostenlos für alle Teilnehmenden der Veranstaltung war. Herzlichen Dank nochmal!
```
